doc/nrf/app_dev/bootloaders_dfu/qspi_xip_split_image.rst

```diff
@@ -215,10 +215,7 @@ Programming with the QSPI XIP split image
 *****************************************
 
 Programming of the application is supported using the :ref:`standard procedure <programming>`.
-The standard procedure will program the firmware using the default nrfjprog configuration which, for QSPI, is PP4IO mode.
-
-.. note::
-      |nrfjprog_deprecation_note|
+The standard procedure will program the firmware using the default nrfutil configuration which, for QSPI, is PP4IO mode.
```

Comment:
```suggestion
The standard procedure programs the firmware using the default nRF Util configuration, which, for QSPI, is the PP4IO mode.
```

Reply:
`nrfutil` is the command line tool so not sure it should be changed to `nRF Util`?

Reply:
Fixed